portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,32 +33,53 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
nit: Can you update this comment and remove the TODO?

Reply:
done